examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2164,18 +2164,14 @@ impl FrontendNode {
         Ok(())
     }
     fn test_grpc_client_unary_method(&mut self) -> TestResult {
-        let grpc_stub = oak::grpc::client::Client::new(
-            "grpc_client",
-            &oak::node_config::grpc_client("https://localhost:7878"),
-            &Label::public_untrusted(),
-        )
-        .map(OakAbiTestServiceClient)
-        .ok_or_else(|| {
-            Box::new(std::io::Error::new(
-                std::io::ErrorKind::Other,
-                "no gRPC client channel available",
-            ))
-        })?;
+        let grpc_stub = oak::grpc::client::init("https://localhost:7878")
```

Comment:
In most cases the clients are long-lived, so the handle leakage is not an issue. These tests are leaking handles and the clients are not long-lived, so perhaps add code to manually close the receivers at the end of the test until we find a better way to resolve it via #1686?

Reply:
There's some manual changes on the way in #1687 (but they may conflict with the changes here)